macros/snowflake_timetravel_table.sql

```diff
@@ -119,12 +123,7 @@
     {{ build_sql }}
   {%- endcall %}
 
-  {{ run_hooks(post_hooks, inside_transaction=True) }}
-
-  -- `COMMIT` happens here
-  {{ adapter.commit() }}
-
-  {{ run_hooks(post_hooks, inside_transaction=False) }}
+  {{ run_hooks(post_hooks) }}
```

Comment:
Interesting that this now reverses the order of post_hooks & commit.  Beforehand, the commit happened after post_hooks, but now it happens before post_hooks.  I would guess the new behavior is more correct anyway.